src/components/transaction-details.tsx

```diff
@@ -20,17 +20,24 @@ import { IconArrowDownRight, IconArrowUpRight } from '@tabler/icons';
 import { BlocksVisualizer } from '@features/blocks-visualizer';
 import { getTransactionStatus } from '@common/utils/transactions';
 import { BtcStxBlockLinks } from '@components/btc-stx-block-links';
+import { useCurrentStxPrice } from '@common/hooks/use-current-prices';
 
+const getUsdPrice = (amountInMicroStacks: number, currentStxPrice: number): string => {
+  const price = (microToStacks(amountInMicroStacks, false) as number) * currentStxPrice;
+  return price < 0.01 ? '<$0.01' : usdFormatter.format(price);
```

Comment:
I think we might use it elsewhere later. It might be worth extracting this formatting logic from the component.